bcda/main.go

```diff
@@ -445,15 +435,9 @@ func createAlphaToken(ttl int, acoSize string) (string, error) {
 	return msg, nil
 }
 
+// This has 100+ references to refactor
```

Comment:
Thought I commented on this in my earlier review, but must not have saved it. I see only ~10 calls for `getEnvInt()`. Moving it altogether would be preferable to keeping this old one around as a wrapper.

Reply:
ok.  I will toss in a ticket for it in this sprint if that is ok.  I think my search included unit test results which was obviously not extra place I need to change this.

Reply:
This is in as BCDA-1128